package.json

```diff
@@ -12,7 +12,7 @@
   "license": "ISC",
   "devDependencies": {
     "pre-commit": "^1.2.2",
-    "prettier": "2.2.1"
+    "prettier": "2.7.1"
```

Comment:
is it really required by your PR?

Reply:
no, it was not required but to format the code i used it.